examples/tokamak-2fluid/2fluid.cxx

```diff
@@ -808,9 +827,9 @@ class TwoFluid : public PhysicsModel {
       
       if (rho_jpar1) {
         if (stagger) {
-          ddt(rho) += SQ(coord->Bxy)*Div_par_CtoL(jpar);
+          ddt(rho) += SQ(coord->Bxy)*Div_par_LtoC(jpar);
         } else  {
-          ddt(rho) += SQ(coord->Bxy)*Div_par(jpar, CELL_CENTRE);
+          ddt(rho) += SQ(coord->Bxy)*Div_par(jpar);
```

Comment:
I think this is always correct here (and the similar constructions above and below)?
```cpp
          ddt(rho) += SQ(coord->Bxy)*Div_par(jpar, CELL_CENTRE);
```

Reply:
It is, and I think we've updated to something like this without the conditionals in `next`. The reason I changed like this was because with the style it's written in, if `stagger = false` then no locations should need to be set (and the `stagger = true` branch is using the `Div_par_LtoC` form). I have no strong opinion on this though, so revert if you like. If I try really hard to justify the change, one could argue that if `stagger = false` then `jpar` should be at `CELL_CENTRE` (and if not it is an error) so putting the `CELL_CENTRE` argument in the `stagger = false` branch could potentially, conceivably, maybe hide an 'error' - I really don't think that's a good reason to have made this change though!